Modules/Quotas/test/Quotas.Application.Tests/Metrics/UsedFileStorageSpaceMetricCalculatorTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Domain.Aggregates.FileMetadata;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Metrics;
+public class UsedFileStorageSpaceMetricCalculatorTests
+{
+    [Fact]
+    public async Task Aggregate_Used_Space_Throws_OverflowException_When_Sum_Wont_Fit_In_A_Long()
+    {
+        // Arrange
+        var repositoryStub = new FileMetadataRepositoryStub(new List<FileMetadata>() { new() { CipherSize = 9223372036854775806 }, new() { CipherSize = 9223372036854775806 } });
```

Comment:
Just pass the number that should be returned
```suggestion
        var repositoryStub = new FileMetadataRepositoryStub(12343554364353);
```

Reply:
I'm assuming that number you put there is not relevant..?

Reply:
No. :) Should have made that more obvious, sorry

Reply:
The thing is I really need to have at least two files. Otherwise, the overflow will not happen. Should I do something like 
` var repositoryStub = new FileMetadataRepositoryStub(12343554364353, 2); `